redhawk/c/c_tree_converter.py

```diff
@@ -81,28 +82,28 @@ def UnescapeEscapeCharacters(self, tree):
   def ConvertFileast(self, tree):
     return N.Module(position = NP.NodePosition(self.filename, 1, 1),
         filename = self.filename,
-        children = map(self.ConvertTree, tree.children()))
+        children = list(map(self.ConvertTree, tree.children())))
```

Comment:
Actually, I think such a function already exists - it's called `ConvertListOfStatements` - is that what you'd like to use?